sdk/include/psdk/ddraw.h

```diff
@@ -3,13 +3,15 @@
 #define __DDRAW_INCLUDED__
 
 #if defined(_WIN32) && !defined(_NO_COM )
-#define COM_NO_WINDOWS_H
-#include <objbase.h>
+  #define COM_NO_WINDOWS_H
+  #include <objbase.h>
 #else
-#define IUnknown void
-#if !defined(NT_BUILD_ENVIRONMENT) && !defined(WINNT)
-        #define CO_E_NOTINITIALIZED 0x800401F0L
-#endif
+  #define IUnknown void
+  #if !defined(NT_BUILD_ENVIRONMENT) && !defined(WINNT)
+    #ifndef CO_E_NOTINITIALIZED
+      #define CO_E_NOTINITIALIZED 0x800401F0L
+    #endif
+  #endif
```

Comment:
Is GCC compilation warning because of non-indentation of these preprocessor commands?

Reply:
No, the one in winerror.h does `_HRESULT_TYPEDEF_(0x800401F0L)` and so the compiler thinks they are different.

The indentation is because that file seems to indent those #ifs.